intro-HPC/ch_FAQ.tex

```diff
@@ -0,0 +1,35 @@
+\chapter{Frequently Asked Questions}
+\label{ch:faq}
+
+\section{When will my job start?}
+
+The clusters use a fair-share sheduling policy. There is no guarantee on when a
+job will start, since it depends on a number of factors. One of these factors is
+the priority of the job, which is determined by
+\begin{itemize}
+    \item historical use: the aim is to balance usage over users, so
+        infrequent users get a highter priority
+
+    \item requested resources (amount of cores, walltime, memory, ...): the larger
+        the resource request is, the lower the priority will be
+
+    \item time waiting in queue: queued jobs get a higher priority over time
+
+    \item user limits: this avoids having a single user hog the entire cluster
+
```

Comment:
Please add:

```
In practice it's impossible to predict when your job(s) will start, since most currently running jobs will finish before their requested walltime expires, new jobs by may be submitted by other users that are assigned a higher priority than your job(s).
```

This is site-specific BTW, so put this whole thing under `\ifgent`. Other sites (e.g. Leuven) do provide the `showstart` command to their users, which gives an estimated start time, but it's basically always wrong, so in Ghent we opted for not providing that to avoid confusion/complaints/etc.

Reply:
Pretty much this same text should maybe be included in another chapter (where we explain how to submit jobs), and then referred to from here?